ext/date/tests/DateTimeZone_serialize_errors.phpt

```diff
@@ -11,5 +11,4 @@ try {
 }
 ?>
 --EXPECTF--
-Warning: DateTimeZone::__unserialize(): Timezone must not contain null bytes in %s on line %d
 Invalid serialization data for DateTimeZone object
```

Comment:
Normal that the warning is suppressed?

If yes you can change the ``--EXPECTF--`` to ``--EXPECT--``